<?xml version="1.0" encoding="UTF-8"?>
<!--
 Generated by Qt.

 Copyright (C) Sapura Secured Technologies, 2016-2024. All Rights Reserved.

 @file
 @version $Id: GisLayerList.ui 1833 2024-03-25 07:55:22Z hazim.rujhan $
-->
<ui version="4.0">
 <class>GisLayerList</class>
 <widget class="QWidget" name="GisLayerList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="layersTitleFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>37</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="mapLayersHLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="mapLayersIconLbl">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/Images/images/icon_mapLayers.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mapLayersLbl">
        <property name="text">
         <string>Map Layers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="floatButton">
        <property name="toolTip">
         <string>Float</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Images/images/icon_float_up.png</normaloff>
          <normalon>:/Images/images/icon_float_down.png</normalon>
          :/Images/images/icon_float_up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toggleBtn">
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Images/images/icon_downward.png</normaloff>
          <normalon>:/Images/images/icon_upward.png</normalon>:/Images/images/icon_downward.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="layerListFrame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_19">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
